optimum/gptq/quantizer.py

```diff
@@ -210,8 +215,13 @@ def convert_model(self, model: nn.Module):
             self.block_name_to_quantize = get_block_name_with_pattern(model)
         block_name = self.block_name_to_quantize
         layers_to_be_replaced = get_layers(model, prefix=block_name)
+        if self.inside_layer_modules is not None:
+            layers_to_keep = sum(self.inside_layer_modules, [])
```

Comment:
I don't understand this

Reply:
We put everything in a single list as we have a list of list.